modules/virtual_network_gateway/variables.tf

```diff
@@ -108,22 +86,37 @@ variable "generation" {
 
 variable "private_ip_address_enabled" {
   description = "Should private IP be enabled on this gateway for connections?"
+  default     = false
   type        = bool
 }
 
+variable "avzones" {
+  description = <<-EOF
+  After provider version 3.x you need to specify in which availability zone(s) you want to place IP.
+
+  For zone-redundant with 3 availability zone in current region value will be:
+  ```["1","2","3"]```
+  EOF
+  default     = []
+  type        = list(string)
```

Comment:
I'm wondering shouldn't we add validation here, either no zones, or all 3? Is that an option? I'm guessing this reflects the PIP as well

Reply:
ok, I added